test/cmd/basicresources.sh

```diff
@@ -92,7 +92,7 @@ echo "pods: ok"
 os::test::junit::declare_suite_end
 
 os::test::junit::declare_suite_start "cmd/basicresources/label"
-os::cmd::expect_success_and_text 'oc create -f examples/hello-openshift/hello-pod.json -o name' 'pod/hello-openshift'
+os::cmd::expect_success_and_text 'oc create -f examples/hello-openshift/hello-pod.json -o name' 'pod.*/hello-openshift'
```

Comment:
Not obvious to me.  What are the characters between pod and slash?

Reply:
removed